notification-service/src/constants/SUBSCRIPTIONS.ts

```diff
@@ -11,15 +11,6 @@ export const SUBSCRIPTIONS: EventSubscription[] = [
                     > Address: [${event.args.addr}](${formatEtherscanLink('address', event.args.addr)})`;
         },
     },
-    {
-        id: 'MCD_DAI_Transfer',
```

Comment:
Do you want to replace it with one on goerli?

Reply:
I have sadly not found a similar contract on goerli that gets called frequently. Therefore I would not add anything at the moment.

Reply:
Can't you just use the same `MCD_DAI` contract? 🙂

Reply:
Otherwise you need to remove `MCD_DAI_Transfer` from the readme (mentioned 2 times). But I still see the reason to move this subscription to `goerli` instead of removing it